docs/fsharp/tour.md

```diff
@@ -1,3 +1,17 @@
+---
+title: Tour of F#
+description: Tour of F# 
+keywords: .NET, .NET Core, F#, Tour
+author: cartermp
+manager: wpickett
+ms.date: 08/18/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 49775139-082e-442f-b5a2-dd402399b5d2
+---
```

Comment:
the metadata being used for F# articles is a bit different. I'll have to change our template to say this.

This is from the Assertions topic for example:
```Markdown
---
title: Assertions (F#)
description: Assertions (F#)
keywords: visual f#, f#, functional programming
author: dend
manager: danielfe
ms.date: 05/16/2016
ms.topic: language-reference
ms.prod: visual-studio-dev14
ms.technology: devlang-fsharp
ms.assetid: 2badaad7-f086-47e7-99c1-91f35117da83 
---
```